scripts/install_keep_core_v1.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
Can we rename the file to `install-v1.sh`?

Reply:
9cd8ce2675c6b0ae21fa1157c0eb01d2a76e6f63